src/main/flight/interpolated_setpoint.c

```diff
@@ -28,25 +28,36 @@
 #include "fc/rc.h"
 #include "flight/interpolated_setpoint.h"
 
+#define PREV_BIG_STEP 1000.0f //threshold for size of jump of packet before the identical data packet
+
 static float setpointDeltaImpl[XYZ_AXIS_COUNT];
-static float prevSetpointDeltaImpl[XYZ_AXIS_COUNT];
 static float setpointDelta[XYZ_AXIS_COUNT];
+static uint8_t holdCount[XYZ_AXIS_COUNT];
 
+typedef struct laggedMovingAverageCombined_u {
```

Comment:
As this is not a union but a struct, the name should be `laggedMovingAverageCombined_s` and not `laggedMovingAverageCombined_u`.

Reply:
Thanks!  Fixed.